ray-operator/controllers/ray/utils/util_test.go

```diff
@@ -90,6 +90,29 @@ func TestCheckName(t *testing.T) {
 	}
 }
 
+func TestCheckRouteName(t *testing.T) {
+	// test 1 -> change
+	str := "cv-traffic-training-202402090958"
```

Comment:
Would you mind structuring the test cases following the usual convention, e.g.:

```
for name, tc := range tests {
		t.Run(name, func(t *testing.T) {
```

Reply:
Updated!